lib/createTopLevelExpect.js

```diff
@@ -1352,159 +1285,182 @@ Unexpected.prototype.lookupAssertionRule = function(
   return null;
 };
 
-function makeExpectFunction(unexpected) {
-  const expect = installExpectMethods(unexpected);
-  unexpected.expect = expect;
-  return expect;
-}
+expectPrototype._assertTopLevelExpect = function(value = true) {
+  if (this !== this._topLevelExpect) {
+    throw new Error('This method only works on the top level expect function');
+  }
+};
 
-Unexpected.prototype.setErrorMessage = function(err) {
+expectPrototype._assertWrappedExpect = function(value = true) {
```

Comment:
Only one small nit: doesn't seem like we use the value argument any more after the method split. Think it can be removed from all callsites too.

Reply:
Ah, my bad, thanks. Fixed in aa3a152614317f7e888b25c8828a49b76b2eca77

Reply:
Nice and np.